### ISSUE TITLE
they changed the captcha style wat do

### ISSUE THREAD

**Issue by HairyMilkshakes** [K1rakishou/Kuroba-Experimental#911]
it's now about half as small so the auto solver fails around 50% of the time.

**Comment by KurubaEX** [comment]
50% of the time? Try 100% of the time. This captcha solver is useless now.

**Comment by K1rakishou** [comment]
Unless someone else trains a new model and releases it to the public I can't do much here because I'm not a ML engineer.

**Comment by XPEH69** [comment]
> Unless someone else trains a new model and releases it to the public I can't do much here because I'm not a ML engineer.

So is the latest update with the "donate solved captcha" setting your effort to create a new model? Even before the 4chan captcha update, the captcha solver barely solved more than 50% of them

**Comment by KurubaEX** [comment]
> > Unless someone else trains a new model and releases it to the public I can't do much here because I'm not a ML engineer.
> 
> So is the latest update with the "donate solved captcha" setting your effort to create a new model? Even before the 4chan captcha update, the captcha solver barely solved more than 50% of them

Didn't know that was an option. I selected yes. Should be set to yes as default for everyone. We need a working captcha solver.

**Comment by HairyMilkshakes** [comment]
They made some headway with the autosolver: https://github.com/drunohazarb/4chan-captcha-solver/issues/1#issuecomment-1673739106

**Comment by yaddy-ranton** [comment]
> They made some headway with the autosolver: [drunohazarb/4chan-captcha-solver#1 (comment)](https://github.com/drunohazarb/4chan-captcha-solver/issues/1#issuecomment-1673739106)

This new model seems to be very accurate. [Yukariin](https://github.com/Yukariin) has posted his model as .tf format. K1rakishou's 4chanCaptchaSolver that was working with KurobaEx seems to be using a [.tflite converted model](https://github.com/K1rakishou/4chanCaptchaSolver/blob/master/app/src/main/assets/chan4_captcha_solver_model.tflite). It seems all that needs to be done is convert the new .tf model to .tflite, update the .tflite file in K1rakishou's repo, and rebuild the project.

**Comment by KurubaEX** [comment]
> > They made some headway with the autosolver: [drunohazarb/4chan-captcha-solver#1 (comment)](https://github.com/drunohazarb/4chan-captcha-solver/issues/1#issuecomment-1673739106)
> 
> This new model seems to be very accurate. [Yukariin](https://github.com/Yukariin) has posted his model as .tf format. K1rakishou's 4chanCaptchaSolver that was working with KurobaEx seems to be using a [.tflite converted model](https://github.com/K1rakishou/4chanCaptchaSolver/blob/master/app/src/main/assets/chan4_captcha_solver_model.tflite). It seems all that needs to be done is convert the new .tf model to .tflite, update the .tflite file in K1rakishou's repo, and rebuild the project.

Thats too complicated for someone like me who has no knowledge of these things. What is a simple way of doing this in kuruba?

**Comment by yaddy-ranton** [comment]
> > > They made some headway with the autosolver: [drunohazarb/4chan-captcha-solver#1 (comment)](https://github.com/drunohazarb/4chan-captcha-solver/issues/1#issuecomment-1673739106)
> > 
> > 
> > This new model seems to be very accurate. [Yukariin](https://github.com/Yukariin) has posted his model as .tf format. K1rakishou's 4chanCaptchaSolver that was working with KurobaEx seems to be using a [.tflite converted model](https://github.com/K1rakishou/4chanCaptchaSolver/blob/master/app/src/main/assets/chan4_captcha_solver_model.tflite). It seems all that needs to be done is convert the new .tf model to .tflite, update the .tflite file in K1rakishou's repo, and rebuild the project.
> 
> Thats too complicated for someone like me who has no knowledge of these things. What is a simple way of doing this in kuruba?

I think it might be easier for @K1rakishou to do it and update his captcha solver. I don't know how to use gradle and I'm running into errors trying to build the captcha solver app. The tflite conversion should be simple enough assuming that the old model that K1rakishou converted and the new model have the same inputs and outputs

**Comment by XPEH69** [comment]
Adamant Cheese updated the captcha solver in his Dev build in like a day. Not sure why Dimon doesn't just implement AC's solver into Ex. Devs solver worked twice as good as Ex's even before the captcha update

**Comment by aardvark0420** [comment]
What is the latest regarding the captcha solver? Mine fails 100 percent of the time, even on newest update. The slide bar does not jump to the correct position anymore. Most of the time it does not even provide alternative solutions to the captcha, but just says "Failed to find the captcha solution".

**Comment by XPEH69** [comment]
> What is the latest regarding the captcha solver? Mine fails 100 percent of the time, even on newest update. The slide bar does not jump to the correct position anymore. Most of the time it does not even provide alternative solutions to the captcha, but just says "Failed to find the captcha solution".

Not sure. Mine had the same issue back then, but I backed up KurobaEx, and then cleared data for both kuroba and the solver and un-installed them. Then I installed KurobaEx, then the solver, and then I restored KurobaEx data. Make sure you delete all your old KurobaEx and solver apks, and re-download them before installing

**Comment by aardvark0420** [comment]
> > What is the latest regarding the captcha solver? Mine fails 100 percent of the time, even on newest update. The slide bar does not jump to the correct position anymore. Most of the time it does not even provide alternative solutions to the captcha, but just says "Failed to find the captcha solution".
> 
> Not sure. Mine had the same issue back then, but I backed up KurobaEx, and then cleared data for both kuroba and the solver and un-installed them. Then I installed KurobaEx, then the solver, and then I restored KurobaEx data. Make sure you delete all your old KurobaEx and solver apks, and re-download them before installing

Ive just deleted both apks and reinstalled both, but the result is the same. When you say clear data for the app and solver, you do mean just to delete the apk, right?

**Comment by XPEH69** [comment]
>Ive just deleted both apks and reinstalled both, but the result is the same. When you say clear data for the app and solver, you do mean just to delete the apk, right?
No, backup your KurobaEx settings, go into system app settings, and clear data/cache for both apps before uninstalling them. Delete all APKs and make sure you download the latest versions of both before installing. Install KurobaEx first, then the solver. Then enable captcha solver in KurobaEx settings

**Comment by aardvark0420** [comment]
> >Ive just deleted both apks and reinstalled both, but the result is the same. When you say clear data for the app and solver, you do mean just to delete the apk, right?
> No, backup your KurobaEx settings, go into system app settings, and clear data/cache for both apps before uninstalling them. Delete all APKs and make sure you download the latest versions of both before installing. Install KurobaEx first, then the solver. Then enable captcha solver in KurobaEx settings

I see. Yes, it seems to be working now! With the arm64-v8a APK version.
